### ISSUE TITLE
Create CityGML importer

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#161]
## Description
See Epic: #160 

## Depends on
#149 

## Acceptance criteria
* Download CityGML data
* Ingest CityGML data into database

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
